### ISSUE TITLE
Unable to serve image with allowed_paths

### ISSUE THREAD

**Issue by bastian110** [gradio-app/gradio#9878]
### Describe the bug

Hello, 
Since, gradio v5 release, i'm unable to display any images from the gradio client.
I tryed the same code with gradio v4 and there no problem.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr


with gr.Blocks() as demo:
    gr.HTML("<img src='/file=image.png' alt='image One'>")

demo.launch(allowed_paths=["image.png"])

```

### Screenshot

![image](https://github.com/user-attachments/assets/83380021-c7f7-4448-98d9-efafbc3f38d9)

### System Info

```shell
pip show gradio 

Name: gradio
Version: 5.4.0
Summary: Python library for easily interacting with trained machine learning models
Home-page: 
Author: 
Author-email: Abubakar Abid <gradio-team@huggingface.co>, Ali Abid <gradio-team@huggingface.co>, Ali Abdalla <gradio-team@huggingface.co>, Dawood Khan <gradio-team@huggingface.co>, Ahsen Khaliq <gradio-team@huggingface.co>, Pete Allen <gradio-team@huggingface.co>, Ömer Faruk Özdemir <gradio-team@huggingface.co>, Freddy A Boulton <gradio-team@huggingface.co>, Hannah Blair <gradio-team@huggingface.co>
License: 
Location: /home/gse1581/playground-rag/.venv/lib64/python3.11/site-packages
Requires: aiofiles, anyio, fastapi, ffmpy, gradio-client, httpx, huggingface-hub, jinja2, markupsafe, numpy, orjson, packaging, pandas, pillow, pydantic, pydub, python-multipart, pyyaml, ruff, safehttpx, semantic-version, starlette, tomlkit, typer, typing-extensions, uvicorn
Required-by:
```

### Severity

I can work around it

**Comment by freddyaboulton** [comment]
HI @bastian110 - the new file route is `/gradio_api/file=...`. This is a duplicate of #9844 #9680
